apps/src/sites/hourofcode.com/pages/views/hoc_events_map_replacement.js

```diff
@@ -13,43 +15,52 @@ $(document).ready(function() {
 
   map.dragRotate.disable();
 
+  // Ensure that if the map is zoomed out such that multiple
+  // copies of the point are visible, the popup appears
+  // over the point clicked on.
+  function getPopupCoordinates(clickLngLat, featureCoordinates) {
+    var normalizedCoordinates = featureCoordinates;
```

Comment:
nit: `'let`

Reply:
oh, are we not going through the preprocessor for this JS file?  In that case, disregard.